docs/docs/server-middleware.md

```diff
@@ -32,6 +32,7 @@ val service = HttpRoutes.of[IO] {
   case GET -> Root / "bad" => BadRequest()
   case GET -> Root / "ok" => Ok()
   case r @ POST -> Root / "post" => r.as[Unit] >> Ok()
+  case r @ POST -> Root / "postText" => r.as[String].flatMap(Ok(_))
```

Comment:
We could also re-use the `reverse` route.

Reply:
I thought about that, but I decided it might be confusing because the logic already includes duplication from a middleware. Adding reversing from the route would make it even more complex, I think.

Reply:
Makes sense. Maybe we should name this route `/echo`, since the response echoes the request?